pages/solid-events.md

```diff
@@ -22,11 +22,16 @@ Solid Events provide an opportunity to meet others who are working on Solid or a
 
 - Have another Solid oriented event to share? Make a pull request to the [solidproject.org events page](https://github.com/solid/solidproject.org/edit/main/pages/solid-events.md).
 
+## 2022 Events
+
+| Date | Event | Organiser | Recordings, Slides and Notes |
+|----|-------|-------------|------------------------------|
+|2022-01-13| Solid World |[Solid Team](https://solidproject.org/team)| Solid World will be back on 13th January 2022. Stay tuned for the Eventbrite link. |
```

Comment:
If you click the "Details" link next to netlify/solidproject-dot-org/deploy-preview (see screenshot below), you can see what this will look like in practice.

![image](https://user-images.githubusercontent.com/4251/144067189-6ecb4590-09b3-4e79-a359-15bd2a3e4885.png)

In this case, it doesn't look quite right yet: https://deploy-preview-656--solidproject-dot-org.netlify.app/events

![image](https://user-images.githubusercontent.com/4251/144067316-78ce3d8c-bf6b-47d9-86fa-3ee780b1892d.png)

I think the solution might just be to add a new line?

Reply:
Thanks a lot!
Fixed it:
![grafik](https://user-images.githubusercontent.com/4144203/144069713-5c7ae4e4-5f89-4f68-b24f-15c2381e1b60.png)